app/controllers/logout_controller.rb

```diff
@@ -0,0 +1,7 @@
+class LogoutController < ApplicationController
```

Comment:
checkout destroy_session_path(:user) from the readme here https://github.com/G5/g5_authenticatable